api/src/app.ts

```diff
@@ -0,0 +1,10 @@
+import * as express from "express"
+import * as cors from "cors"
+import statusRouter from "./routes/status"
+
+const app: express.Application = express()
+app.use(cors())
+
+app.use("/status", statusRouter)
```

Comment:
If we rename it to `/health` we can then use it as a health endpoint for our ecs services.